### PR TITLE
Phishing playbook readme update for campaign

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
**In-Progress** - requires new changes after campaign rework

## Related Issues
Related to: https://github.com/demisto/etc/issues/33810

## Description
- Added changes to the pack and playbook readmes regarding phishing campaigns.

## Minimum version of Cortex XSOAR
6.0.0

## Does it break backward compatibility?
No

## Must have
Documentation

### CODE REVIEW

**Comment by kirbles19** [comment]
@idovandijk Looks good

**Review by michalgold — APPROVED** [review]

**Comment by idovandijk** [comment]
I won't merge until we merge the campaign stuff

**Pull request merged**
